api/class.templatemethods.php

```diff
@@ -472,8 +472,7 @@ public function method_strftime($mData, $aParams)
         }
 
         return strftime(
-            $sFormat,
-            intval(trim($this->templateDataAsString($mData)))
+            $sFormat, (int)trim($this->templateDataAsString($mData))
```

Comment:
Please correct the the CS. Only one argument is allowed per line in a multi-line function call.

Reply:
Done.